analytics-webapps/src/main/webapp/vue-app/breadcrumb-portlet/initComponents.js

```diff
@@ -0,0 +1,9 @@
+import Breadcrumb from './components/AnalyticsDashboardBreadcrumb.vue';
+
+const components = {
+    'bread-crumb-analytics': Breadcrumb,
```

Comment:
it's better to define Vue modules like we name the Java packages, from generic to specific. IMO to change to : 
```suggestion
    'analytics-breadcrumb': Breadcrumb,
```